lib/pymedphys/docs/howto/advanced/documentation.rst

```diff
@@ -1,10 +1,6 @@
 Writing documentation
 =====================
 
-.. contents::
-    :local:
-    :backlinks: entry
-
 This documentation site uses a toolbox called Sphinx. This particular
```

Comment:
Are we still using Sphinx?  Maybe the title of the PR should be
"Swap to Jupyter Book from XYZ" 
and if XYZ is Sphinx... then this file needs to be tweaked.

Reply:
See above comment.